dataland-frontend/src/components/ui/EuTaxoSearchBar.vue

```diff
@@ -1,68 +1,241 @@
 <template>
   <MarginWrapper>
     <div class="grid">
-      <div class="col-8">
-      <span class="p-fluid">
-        <span class="p-input-icon-left ">
-            <i class="pi pi-search" aria-hidden="true"/>
-            <InputText type="text" v-model="model" placeholder="Search by company name, LEI, PermID or ISIN"
-                       disabled="true" name="eu_taxonomy_search_input" class="h-3rem"/>
+      <div class="col-8 text-left" v-if="!scrolled">
+        <span class="p-fluid" >
+          <span class="p-input-icon-left p-input-icon-right ">
+              <i class="pi pi-search" aria-hidden="true" style="z-index:20; color:#958D7C"/>
+                    <i v-if="loading" class="pi pi-spinner spin" aria-hidden="true" style="z-index:20; color:#958D7C"/>
+                    <i v-else aria-hidden="true"/>
+              <AutoComplete v-model="selectedCompany" :suggestions="filteredCompaniesBasic" @focus="focused" @focusout="unfocused"
+                            @complete="searchCompany($event)" placeholder="Search a company by name"
+                            inputClass="h-3rem" ref="cac"
+                            field="companyName" style="z-index:10" name="eu_taxonomy_search_input"
+                            @keyup.enter="filter=true; table=true; $router.push({name: 'Search Eu Taxonomy', query: {input: selectedCompany}}); queryCompany(); close();"
```

Comment:
diese methoden können in script als eine Methode defeniert werden

Reply:
wurde so gemacht, waren nur für das entwickeln direkt in dem element